### PR TITLE
feat: added wishlist

### PR DESCRIPTION
**Pull request by Rohitprasad83**
# wishlist page
https://games-kart-preview.netlify.app/pages/wishlist.html

### CODE REVIEW

**Review by Naveenchandar — APPROVED** [review]
Looks good

File: style/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+.card__buttons {
+    justify-content: center;
+}
+.card__buttons .btn {
+    width: 100%;
+}
+
+.main__container {
+
```
Comment (ishitakeshawani) [inline style/wishlist.css]: remove extra space from here

File: style/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    border-radius: 1rem;
+    padding: 2rem;
+}
+.wishlist {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+@media only screen and (max-width: 640px) {
```
Comment (ishitakeshawani) [inline style/wishlist.css]: You can use ems for media queries

**Review by ishitakeshawani — CHANGES_REQUESTED** [review]

File: style/wishlist.css (hunk 2 above)
Comment (Rohitprasad83) [reply style/wishlist.css]: Thanks, I have changed the unit to em from pixel.

File: style/wishlist.css (hunk 1 above)
Comment (Rohitprasad83) [reply style/wishlist.css]: Applied Prettier code formatting to removed unwanted space.

**Review by ishitagecbvn — APPROVED** [review]
LGTM

**Review by ishitakeshawani — APPROVED** [review]

**Pull request merged**